### PR TITLE
 Cluster-reader should view all of the openshift logs on kibana

### PR DESCRIPTION
**Pull request by mburke5678**
https://bugzilla.redhat.com/show_bug.cgi?id=1624338

### CODE REVIEW

**Comment by mburke5678** [comment]
@jcantrill 
> *modify verbiage to state that 'operations users' are those who can satisfy the SubjectAccessReview mentioned in #c.

I am not clear on what the SubjectAccessReview is. Is this an object that the user creates at some point or is it something we create? We don't document SAR at all (save for references in the API guide).

File: install_config/aggregate_logging.adoc [hunk 1]
```diff
@@ -533,7 +530,8 @@ this mode, their saved queries, visualizations, and dashboards are not shared.
 
 You may also set the value `shared_ops`. In this mode, all operations users
 share a Kibana index which allows each operations user to see the same
-queries, visualizations, and dashboards.
+queries, visualizations, and dashboards. Operations users are those users 
+who can satisfy the `SubjectAccessReview`.
```
Comment (jcantrill) [inline install_config/aggregate_logging.adoc]: ... ` oc auth can-i get pod/logs -n default`

Comment (jcantrill) [reply install_config/aggregate_logging.adoc]: @mburke5678 Maybe we drop the ref to `SubjectAccessReview` and just define the command.  It is something they can run to determine there role.

Comment (mburke5678) [reply install_config/aggregate_logging.adoc]: @jcantrill What does the user do if the command shows:
```
#oc auth can-i get pod/logs -n default
no
```
Does the user need `cluster-admin` and `cluster-reader`? You seem unsure in the BZ. What can we tell users to do?

Comment (jcantrill) [reply install_config/aggregate_logging.adoc]: They need an administrator to grant those permissions which is typically some cluster admin role.  I don't know if we want to define the here or not.  It implies elevated permissions

**Comment by mburke5678** [comment]
@jcantrill PTAL

**Review by jcantrill — APPROVED** [review]

**Comment by mburke5678** [comment]
@xltian PTAL

**Comment by xltian** [comment]
@QiaolingTang will review from QE.

**Comment by QiaolingTang** [comment]
LGTM

File: install_config/aggregate_logging.adoc [hunk 2]
```diff
@@ -528,12 +525,19 @@ For example, the selector could be `node-type=infrastructure`. At least
...
 share a Kibana index which allows each operations user to see the same
-queries, visualizations, and dashboards.
+queries, visualizations, and dashboards. To determine if you are an operations user:
+
+----
+#oc auth can-i view pod/logs -n default
+yes
+----
+
+If you do not have proper access, contact your cluster administrator.
```
Comment (kalexand-rh) [inline install_config/aggregate_logging.adoc]: s/If you do not have proper access/If you cannot view the pod logs,
IBM Style says:
```
proper: Do not use to mean “correct,” “applicable,” or “appropriate.”
```

**Review by kalexand-rh — COMMENTED** [review]
One style pick, otherwise LGTM!

**Pull request merged**